api/src/main/java/marquez/MarquezApp.java

```diff
@@ -72,6 +73,10 @@ public void initialize(@NonNull Bootstrap<MarquezConfig> bootstrap) {
             new EnvironmentVariableSubstitutor(ERROR_ON_UNDEFINED)));
 
     bootstrap.getObjectMapper().disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+
+    bootstrap.addBundle(
+        new AssetsBundle(
+            "/assets", "/api/v1/graphql-playground", "index.htm", "graphql-playground"));
```

Comment:
Similar comment as below, but I'd suggest using the mapping `/graphql-playground` at the root-level for the graphql playground.